src/components/App.tsx

```diff
@@ -16,16 +16,38 @@ import DependenciesView from 'components/DependenciesView';
 import { checkUserStatus } from 'actions/appLoad';
 import Board from 'components/Board';
 import { SquareSpinner } from 'styles/ThemeComponents';
+import WebSocketsService from 'utils/WebSocketsService';
 
 toast.configure({ hideProgressBar: true });
 
+const messageTypeToActionTypeMap = {
+  Board: 'OPENED_BOARD',
+  BoardList: 'OPENED_BOARD_LIST',
+};
+
+const createReceiveWsAction = (type, data) => ({
+  type,
+  subType: 'SUCCESS',
+  payload: { success: data },
+});
+
 export function App() {
   const myState = useSelector<IStoreState, IMyState>(state => state.myState);
   const history = useHistory();
   const dispatch = useDispatch();
   const actions = bindActionCreators<{}, ActionCreatorsMapObject>({ checkUserStatus }, dispatch);
+
+  const onMessageCallback = event => {
+    const data = JSON.parse(event.data);
+    const type = messageTypeToActionTypeMap[data['@type']];
+    dispatch(createReceiveWsAction(type, data));
+  };
+
   useEffect(() => {
-    actions.checkUserStatus(history);
+    actions.checkUserStatus(history).then(() => {
+      WebSocketsService.initWebSocketConnection();
+      WebSocketsService.handleMessage(onMessageCallback);
```

Comment:
```suggestion
      WebSocketsService.handleMessage(onMessageCallback);
      WebSocketsService.initWebSocketConnection();
```
This just makes it clearer that the handler must be set before the promise returned by `init` resolves.

Reply:
Looks good 👍